aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -134,6 +187,13 @@ async fn main() -> anyhow::Result<()> {
                 .with_context(|| "Failed to generate release proposal scripts".to_string())?;
             Ok(())
         },
+        Commands::SimulateMultiStepProposal {
+            network,
+            proposal_dir,
+        } => {
+            simulate_multistep_proposal(network.to_url()?, &proposal_dir).await?;
```

Comment:
Yea agreed. The CI job will most likely just take a yaml file/

Reply:
Okay, I did it in a slightly different way -- you can now add an optional argument `--simulate <network>` to the `generate-proposals` command to do simulation after generation. Does you guys think this would work?

I'm thinking that if I don't do this, then I'll need to add `release_config` as an optional argument to the simulate command. Which one do you think would have the better UX?

Reply:
Optional sounds reasonable to me. You can spawn a temp folder for this.

Reply:
Sorry if I didn't make my question clear. Was asking you guys whether you prefer having
```
aptos-release-builder generate-proposals .. --simulate mainnet
```
or
```
aptos-release-builder simulate-multistep-proposal .. --release-config data/release.yaml
```

Reply:
I think the --simulate works well is pretty intuitive